test/integration/data-types.test.js

```diff
@@ -145,6 +145,25 @@ describe(Support.getTestDialectTeaser('DataTypes'), function() {
       return testSuccess(Type, [1, 2]);
     });
   }
+  
+  if (dialect === 'mssql') {
+    it.only('should parse TIMESTAMP field', function () {
```

Comment:
Thank you for picking that up. I will remove this; I was doing some final debugging and missed that out.